tapioca/adapters.py

```diff
@@ -81,6 +82,14 @@ def get_iterator_next_request_kwargs(self, iterator_request_kwargs,
                                          response_data, response):
         raise NotImplementedError()
 
+    def _is_token_expired(self, error):
```

Comment:
In the ``TapiocaAdapter`` this should raise ``NotImplementedError``, but lets provide a mixin that makes the verification ``error.status_code == 401``